api/go.mod

```diff
@@ -11,8 +11,8 @@ retract v1.28.0 // tag was mutated
 
 require (
 	github.com/google/go-cmp v0.5.9
-	github.com/hashicorp/consul/proto-public v0.5.1
-	github.com/hashicorp/consul/sdk v0.15.0
+	github.com/hashicorp/consul/proto-public v0.6.1
+	github.com/hashicorp/consul/sdk v0.16.0
```

Comment:
sdk looks like its at 0.16.0 and 0.16.1 in different parts of this PR, would we want to standardize on one?

Reply:
good call.  1.16.0 is the correct max which means the submodules are correct but these root and test* go.mods should be 1.16.0

Reply:
this is fixed in e9d89f205be9ede7fb9e37207af41b0199b98606

thank you for calling this out.